### PR TITLE
Moving create vpn config to script

### PR DESCRIPTION
**Pull request by ben851**
## What happens when your PR merges?
Moving the auto generation of vpn config to a script and suppressing error outputs to be safe.

## What are you changing?
- [ ] Releasing a new version of Notify
- [ ] Changing kubernetes configuration
- [x] Github Workflows

## Provide some background on the changes
Part of VPN reconfig

## Checklist if making changes to Kubernetes:
- [x] I know how to get kubectl credentials in case it catches on fire

## After merging this PR
- [ ] I have verified that the tests / deployment actions succeeded
- [ ] I have verified that any affected pods were restarted successfully
- [ ] I have verified that I can still log into [Notify production](https://notification.canada.ca)
- [ ] I have verified that the smoke tests still pass on production
- [ ] I have communicated the release in the #notify Slack channel.

### CODE REVIEW

**Review by P0NDER0SA — APPROVED** [review]
Nice.  Much cleaner

File: .github/workflows/helmfile_production_apply.yaml [hunk 1]
```diff
@@ -58,19 +58,7 @@ jobs:
...
-          CERT=$(terragrunt output --raw gha_vpn_certificate)
-          KEY=$(terragrunt output --raw gha_vpn_key)
-          aws ec2 export-client-vpn-client-configuration --client-vpn-endpoint-id $ENDPOINT_ID --output text > /var/tmp/production.ovpn
-          echo "<cert>
-          $CERT
-          </cert>" >> /var/tmp/production.ovpn
-          echo "<key>
-          $KEY
-          </key>" >> /var/tmp/production.ovpn        
+          scripts/createVPNConfig.sh production 2> /dev/null
```
Comment (jimleroyer) [inline .github/workflows/helmfile_production_apply.yaml:61]: ❤️

File: scripts/createVPNConfig.sh [hunk 2]
```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+# This script will create a VPN configuration file for the specified environment
+# Usage: ./createVPNConfig.sh <environment>
+# Example: ./createVPNConfig.sh staging
+ENVIRONMENT=$1
+git clone https://github.com/cds-snc/notification-terraform.git /var/tmp/notification-terraform
```
[the remark this answers is not in the document]
Comment (ben851) [reply scripts/createVPNConfig.sh:6]: Technically correct but not required.

File: scripts/createVPNConfig.sh [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+cd /var/tmp/notification-terraform/env/$ENVIRONMENT/eks
+export INFRASTRUCTURE_VERSION=$(cat ../../../.github/workflows/infrastructure_version.txt)
```
[the remark this answers is not in the document]
Comment (ben851) [reply scripts/createVPNConfig.sh:8]: Makes senses but not required.

File: scripts/createVPNConfig.sh [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+ENDPOINT_ID=$(terragrunt output --raw gha_vpn_id)
```
[the remark this answers is not in the document]
Comment (ben851) [reply scripts/createVPNConfig.sh:9]: Technically correct but it will fail regardless, so not required.

**Pull request merged**